### PR TITLE
Fix save_artifact on manifest image and skip VA scan on manifest

### PR DESCRIPTION
**Pull request by pbaity**
This PR cleaned up whitespace that my code formatter was complaining about, so when viewing the PR you can select to not view whitespace changes to narrow down what was changed.

Earlier I had moved the `save_artifact` on the manifest image to the `release` stage since the VA scan was trying to scan the manifest and failing. This was more complicated though and was failing repeatedly. So instead I decided to add another property to the artifact named `manifest` which is either true or false; it's set to false for all the images except the manifest. Then in the VA scan stage I check for this property to skip scanning the manifest.

I also fixed which API key is used in the CD pipeline for skopeo operations - while testing the staging key was used both for `src` and `dest`, but `dest` should use the production API key.

### CODE REVIEW

File: .one-pipeline.yaml [hunk 1]
```diff
@@ -158,8 +158,8 @@ containerize:
      DIGEST="$(skopeo inspect docker://$IMAGE | grep Digest | grep -o 'sha[^\"]*')"
      #ARCH="$(echo $i | grep -o '\(amd64\|s390x\|ppc64le\)$' || echo 'amd64')"
      ARCH=amd64
-     echo "Saving artifact bundle-$i name=$IMAGE digest=$DIGEST"
-     save_artifact bundle-$i type=image name="$IMAGE" "digest=$DIGEST" "arch=$ARCH"
+     echo "Saving artifact bundle-$i name=$IMAGE digest=$DIGEST manifest=$MANIFEST"
```
Comment (mcurran-us) [inline .one-pipeline.yaml:161]: MANIFEST hasn't been set yet.

File: .one-pipeline.yaml [hunk 2]
```diff
@@ -169,8 +169,8 @@ containerize:
      DIGEST="$(skopeo inspect docker://$IMAGE | grep Digest | grep -o 'sha[^\"]*')"
      #ARCH="$(echo $i | grep -o '\(amd64\|s390x\|ppc64le\)$' || echo 'amd64')"
      ARCH=amd64
-     echo "Saving artifact catalog-$i name=$IMAGE digest=$DIGEST"
-     save_artifact catalog-$i type=image name="$IMAGE" "digest=$DIGEST" "arch=$ARCH"
+     echo "Saving artifact catalog-$i name=$IMAGE digest=$DIGEST manifest=$MANIFEST"
```
Comment (mcurran-us) [inline .one-pipeline.yaml:172]: Same comment as above.

File: .one-pipeline.yaml (hunk 1 above)
Comment (pbaity) [reply .one-pipeline.yaml]: Fixed!

File: .one-pipeline.yaml (hunk 2 above)
Comment (pbaity) [reply .one-pipeline.yaml]: Fixed!

**Review by mcurran-us — APPROVED** [review]

**Pull request merged**